src/main/java/io/github/thebusybiscuit/mobcapturer/MobCapturer.java

```diff
@@ -169,6 +179,12 @@ public void onEnable() {
         register("Blaze", EntityType.BLAZE, new StandardMobAdapter<>(Blaze.class), "533acae6e075a578ccfc7dc2d5a15dbccfa8f59c609f9703889ef54c742c56");
         register("Magma Cube", EntityType.MAGMA_CUBE, new SlimeAdapter<>(MagmaCube.class), "1185657c38acdd8f95e1d2cd1115bb0f11139ad2b3ce442267e69706d916e");
         register("Ghast", EntityType.GHAST, new StandardMobAdapter<>(Ghast.class), "c442c228f099fdfc1c6b46dfc80b252d81f7fb1739deb16ee7a597c17f7c9");
+        register("Piglin", EntityType.PIGLIN, new PiglinAdapter(), "4e2ce3372a3ac97fdda5638bef24b3bc49f4facf751fe9cad645f15a7fb8397c");
+        register("Piglin Brute", EntityType.PIGLIN_BRUTE, new PiglinBruteAdapter(), "4e2ce3372a3ac97fdda5638bef24b3bc49f4facf751fe9cad645f15a7fb8397c");
+        register("Zombified Piglin", EntityType.ZOMBIFIED_PIGLIN, new ZombifiedPiglinAdapter(), "4e2ce3372a3ac97fdda5638bef24b3bc49f4facf751fe9cad645f15a7fb8397c");
+        register("Hoglin", EntityType.HOGLIN, new HoglinAdapter(), "4e2ce3372a3ac97fdda5638bef24b3bc49f4facf751fe9cad645f15a7fb8397c");
+        register("Zoglin", EntityType.ZOGLIN, new StandardMobAdapter<>(Zoglin.class), "4e2ce3372a3ac97fdda5638bef24b3bc49f4facf751fe9cad645f15a7fb8397c");
+        register("Strider", EntityType.STRIDER, new StriderAdapter(), "4e2ce3372a3ac97fdda5638bef24b3bc49f4facf751fe9cad645f15a7fb8397c");
```

Comment:
Why do these have missing textures?

Reply:
as stated before, I don't know how to create the textures so it was made as a stand-in.

Reply:
Didn't you have textures before though?

Reply:
Nope they were also missing textures, it is also written in the PR itself.